src/components/Note/Note.tsx

```diff
@@ -71,9 +74,19 @@ const Note = (props: NoteProps) => {
               <figcaption className="note__author-name">{props.authorName}</figcaption>
             </figure>
           )}
-          <Votes className="note__votes" noteId={props.noteId!} votes={props.votes.concat(props.childrenNotes.flatMap((n) => n.votes))} activeVoting={props.activeVoting} />
+          <Votes
+            className="note__votes"
+            noteId={props.noteId!}
+            votes={filteredVotes.concat(
+              filterVotes(
+                props.childrenNotes.flatMap((n) => n.votes),
+                props.activeVoting
+              )
+            )}
+            activeVoting={props.activeVoting}
+          />
         </footer>
-        <NoteDialog {...props} onClose={handleShowDialog} show={showDialog} />
+        <NoteDialog {...{...props, votes: filteredVotes}} onClose={handleShowDialog} show={showDialog} />
```

Comment:
```suggestion
        <NoteDialog {...props} votes={filteredVotes} onClose={handleShowDialog} show={showDialog} />
```
Don't know if this work but looks more beautiful, doesn't it?

Reply:
Works. Good to know.